src/content/en/fundamentals/performance/rail.md

```diff
@@ -132,21 +142,16 @@ To evaluate your site against RAIL metrics, use the Chrome DevTools [Timeline to
     <tr>
       <td data-th="RAIL Step"><strong>Response</strong></td>
       <td data-th="Key Metric">Input latency (from tap to paint) < 100ms.</td>
-      <td data-th="User Test">User taps on an icon or button (for example, opening the nav menu, tapping Compose).</td>
+      <td data-th="User Test">User taps a button (for example, opening navigation).</td>
     </tr>
     <tr>
       <td data-th="RAIL Step"><strong>Response</strong></td>
-      <td data-th="Key Metric">Input latency (from tap to paint) < 16ms.</td>
-      <td data-th="User Test">User drags their finger and the app's response is bound to the finger position (for example, pull to refresh, swiping a carousel).</td>
-    </tr>
-    <tr>
-      <td data-th="RAIL Step"><strong>Animation</strong></td>
       <td data-th="Key Metric">Input latency (from tap to paint) < 100ms for initial response.</td>
       <td data-th="User Test">User initiates page scroll or animation begins.</td>
     </tr>
```

Comment:
I changed the metric above from Animation to Response. I think the current published version is a mistake.

Here's the original message from @tdresser that prompted these changes:

The response row says from tap to paint (which should have 100ms as the threshold) on the left, but then talks about drag effects (which should have ~10ms as the threshold) on the right.
The animation row says from tap to paint (with the correct 100ms threshold) on the left, but then talks about scroll and animation start on the right, which are somewhat different.